exercises/spiral-matrix/example.rs

```diff
@@ -0,0 +1,42 @@
+pub fn get_spiral(size: usize) -> Vec<Vec<usize>> {
+    let mut matrix: Vec<Vec<usize>> = Vec::with_capacity(size);
+    for i in 0..size {
```

Comment:
There's a shorthand for initializing a vector with `N` copies of type `T`: `vec![T; N]`. You could use it here to avoid the loop: `vec![vec![0; size]; size]`.

I'm not certain, but I suspect that the compiler can optimize array initialization of this form to something quicker than a nested loop.